modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/SideMenu/utils/navigationIcons.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import * as NavigationMenuIcons from '../../../../../common/icons/navigation-menu';
+
+const navigationIcons = {
+	analytics: [
+		NavigationMenuIcons.AnalyticsIcon,
+		NavigationMenuIcons.AnalyticsIconGray,
+	],
+	commerce: [
+		NavigationMenuIcons.CommerceIcon,
+		NavigationMenuIcons.CommerceIconGray,
+	],
+	dxp: [NavigationMenuIcons.DXPIcon, NavigationMenuIcons.DXPIconGray],
```

Comment:
This should be made consistent across each key value of the object.  For this case, we should make new lines for each array element.  It would be easier to read.

Reply:
DXP, LXC, productActivation are single lined.

Reply:
This is a behavior caused by the format, otherwise it would fail the test. Thanks!